src/components/Output.js

```diff
@@ -9,16 +11,21 @@ export function Output({
   updateBabelConfig,
   removeBabelConfig,
 }) {
-  return babelConfig.map((config, index) => {
-    return (
-      <CompiledOutput
+  let panes = babelConfig.map((config, index) => {
+
+    console.log(index)
+
+    return {
+      menuItem: 'Config ' + index, render: () => <CompiledOutput
         source={debouncedSource}
         customPlugin={enableCustomPlugin ? customPlugin : undefined}
         config={config}
         key={index}
         onConfigChange={config => updateBabelConfig(config, index)}
         removeConfig={() => removeBabelConfig(index)}
       />
```

Comment:
Do we want the `Config` tab width to take up the entire width, or leave with the current (taking up about half)?

![image](https://user-images.githubusercontent.com/22751307/89829684-fc857180-db28-11ea-8e09-ca2f94095380.png)

Reply:
Lol, yeah it should be 100% I think.